tests/base/src/test/java/org/wildfly/security/http/impl/AbstractBaseHttpTest.java

```diff
@@ -358,6 +358,9 @@ public boolean resumeRequest() {
         }
 
         public HttpScope getScope(Scope scope) {
+            if (requestURI != null && "/clientApp/logout/callback".equals(requestURI.getPath())){
+                return null;
+            }
```

Comment:
Why is this needed here?

Reply:
This is needed to allow assertUserNotAuthenticated() after user logout to function properly.

Reply:
Just wondering if this is the right place to add this logic since getScope doesn't take the requestURI as a param.

Reply:
done